t/27-consoles-s3270.t

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/perl
+
+use Data::Dumper;
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+use Test::Mock::Time;
+use Test::MockModule;
+use Test::MockObject;
+use Test::Output qw(combined_like stderr_like);
+use Test::Warnings qw(:all :report_warnings);
+use POSIX qw(waitpid _exit);
+use FindBin '$Bin';
+use lib "$Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '5';
+use consoles::console;
+use backend::ipmi;
+use consoles::VNC;
+use consoles::s3270;
+use cv;
+
+cv::init;
+require tinycv;
+
+$bmwqemu::topdir = "$Bin/..";
+$bmwqemu::vars{ZVM_HOST} = "localhost";
+$bmwqemu::vars{ZVM_GUEST} = "guest";
+$bmwqemu::vars{ZVM_PASSWORD} = "password";
+
+my $ipc_run_mock = Test::MockModule->new('IPC::Run');
+my $vnc_mock = Test::MockModule->new('consoles::VNC');
+my $inet_mock = Test::MockModule->new('IO::Socket::INET');
+my $s = Test::MockObject->new->set_true(qw(sockopt fileno print connected close blocking));
+# $s->mock(read => sub { $_[1] = $s->mocked_read; length $_[1] });
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details